bottlecap/src/tags/lambda/tags.rs

```diff
@@ -113,6 +116,16 @@ fn tags_from_env(
     if let Ok(memory_size) = std::env::var(MEMORY_SIZE_VAR) {
         tags_map.insert(MEMORY_SIZE_KEY.to_string(), memory_size);
     }
+    if let Ok(exec_runtime_env) = std::env::var(RUNTIME_VAR) {
+        // AWS_Lambda_java8
```

Comment:
if this takes ~25ms can we add an opt-out for people who may know what their runtime is and not care?

Reply:
it seems the 25ms delay is coded in goagent (more details in the jira https://datadoghq.atlassian.net/browse/SVLS-4904 ) so we are not adding anything here